docker-for-aws/release-notes.md

```diff
@@ -6,41 +6,89 @@ title: Docker for AWS Release Notes
 
 {% include d4a_buttons.md %}
 
-## 1.13.0-1
-Release date: 1/18/2017
+## Stable Channel
+
+### 1.13.1-2 (stable)
+Release date: 02/08/2017
 
 {{aws_button_latest}}
 
-### New
+**New**
```

Comment:
I don't mind changing these from H-level headings to just bold text, but it means that you can't permalink straight to the new features for each version if you wanted to. Up to you!